db/migrate/20200117130357_journalist_has_many_articles.rb

```diff
@@ -0,0 +1,5 @@
+class JournalistHasManyArticles < ActiveRecord::Migration[6.0]
```

Comment:
This is not a blocker by the naming of this migration stands out. The modification is done to the Articles table and not Journalist. so I would have gone with `AddJournalistToArticles`

Reply:
i agree, makes sense

Reply:
not fixed